pkg/dyninst/procsubscribe/procscan/start_time_cache.go

```diff
@@ -0,0 +1,78 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package procscan
+
+// startTimeCache caches process start times keyed by PID.
+//
+// It stores ticks since boot, and uses the high bit as a "seen in this scan"
+// marker.
+//
+// getStartTime marks any accessed PID as seen. sweep keeps seen entries
+// (clearing the marker) and deletes unseen ones (PIDs not observed in procfs).
+//
+// The cache is bounded by maxSize. When full, inserting evicts one entry by
```

Comment:
Why bother with this? It'd be simpler to not insert at all and rely on the sweeping reclaim space.